Controller/AbstractFOSRestController.php

```diff
@@ -12,15 +12,22 @@
 namespace FOS\RestBundle\Controller;
 
 use FOS\RestBundle\View\ViewHandlerInterface;
-use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Contracts\Service\ServiceSubscriberInterface;
 
+$ref = new \ReflectionMethod(ServiceSubscriberInterface::class, 'getSubscribedServices');
+
+// Has the ServiceSubscriberInterface a return type hint
+if ($ref->getReturnType() !== null) {
```

Comment:
```suggestion
if (null !== $ref->getReturnType()) {
```

... to fix the style check